sync-version/action.yaml

```diff
@@ -39,7 +42,23 @@ runs:
       env:
         SNAP_NAME: ${{ github.event.repository.name }}
       run: |
-        version="$(cat snap/snapcraft.yaml | yq -r '.version')"
+        if [[ -n "${{ inputs.snapcraft-yaml-path }}"; then
+            yaml_path="${{ inputs.snapcraft-yaml-path }}"
+        else
+            snapcraft_yaml_paths=(
+              "snap/snapcraft.yaml"
+              "snapcraft.yaml"
+              "build-aux/snap/snapcraft.yaml"
+              ".snapcraft.yaml"
+              )
+              
+        for file in "${snapcraft_yaml_paths[@]}"; do
+            if [[ -f "$file" ]]; then
+                yaml_path="$file"
+            fi
+        done
+        fi
+        version="$(cat $yaml_path | yq -r '.version')"
```

Comment:
@jnsgruk I'm not convinced it's a good idea to add the version in the commit message. Depending on the `update-script`, the application version might not even have changed. Can't this workflow just say `chore: update ${SNAP_NAME}`?

Reply:
Also there can be snaps that'll use appstream metadata to adopt versions, those will never have the version line. What about those?

Reply:
Personally I think it's important - if nothing else for going back in time and understanding what happened when we get bugs reported etc.

Reply:
I have added a commit, I think we can use it for now. Please check though.